src/main/io/serial_cli.c

```diff
@@ -358,8 +363,8 @@ static const char * const lookupTableGPSSBASMode[] = {
 };
 #endif
 
-static const char * const lookupTableCurrentSensor[] = {
-    "NONE", "ADC", "VIRTUAL"
+static const char * const lookupTableCurrentMeter[] = {
```

Comment:
yes, i will re-order them since most boards do not have built in amperage meters.